CalPatRec/src/AgnosticHelixFinder_module.cc

```diff
@@ -729,8 +730,6 @@ namespace mu2e {
   //-----------------------------------------------------------------------------
   void AgnosticHelixFinder::tcHitsFill(size_t tc) {
 
-    size_t sortStartIndex = 0;
-
     cHit hit;
     hit.circleError2 = 1.0;
```

Comment:
Please move these initializations to the cHit declaration.  All the data members should always be initialized to a default value

Reply:
done